core/build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
     api "com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}"
     api group: 'com.google.code.gson', name: 'gson', version: '2.8.9'
     api group: 'com.tdunning', name: 't-digest', version: '3.3'
+    api group: 'org.opensearch', name: 'opensearch', version: "${opensearch_version}"
```

Comment:
Core module should not directly depend on OpenSearch library.

Reply:
Removed the dependency and moved formatting logic inside `OpenSearchDateType` class as discussed offline